engine/src/main/java/org/terasology/rendering/dag/FBOTask.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2016 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.rendering.dag;
+
+import static org.lwjgl.opengl.EXTFramebufferObject.GL_FRAMEBUFFER_EXT;
+import static org.lwjgl.opengl.EXTFramebufferObject.glBindFramebufferEXT;
+
+/**
+ *
+ */
+final class FBOTask extends AbstractTask<Integer> {
+    FBOTask(Object value) {
+        super(value);
+    }
+
+    @Override
+    public void execute() {
+        glBindFramebufferEXT(GL_FRAMEBUFFER_EXT, value);
+    }
```

Comment:
Now, this is an interesting one. Here what I'd like is 

`glBindFramebufferEXT(GL_FRAMEBUFFER_EXT, fboToBind);`

But the abstract super class imposes on you to use "value" instead, which is a generic and largely meaningless name. 

To me this would be a good enough reason to ditch the abstract class.